src/javascript/crypto/e2e/extension/launcher.js

```diff
@@ -104,6 +105,11 @@ ext.Launcher.prototype.start_ = function(passphrase) {
           'Version',
           manifest.name + ' v' + manifest.version);
         }
+      }, function(e) {
+        this.updatePassphraseWarning();
+        if (!(e instanceof e2e.openpgp.error.PassphraseError)) {
+          throw e;
```

Comment:
makes sense. will keep `updatePassphraseWarning()` and let it be thrown unconditionally